manifests/params.pp

```diff
@@ -6,7 +6,7 @@
     warning("${facts['os']['family']} is not supported")
   }
 
-  $kafka_version  = '2.4.1'
+  $kafka_version  = '2.7.0'
```

Comment:
This is more than just a modulesync. That should be noted in somewhere in the changelog. The best way to do that is to open an issue for it.

Reply:
I reverted this change and simply using the "apache archive", which will make sure that the test will continue to run in the future.